staging/src/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -104,17 +105,14 @@ func WithExponentialBackoff(initialBackoff time.Duration, webhookFn func() error
 	var err error
 	wait.ExponentialBackoff(backoff, func() (bool, error) {
 		err = webhookFn()
-		// these errors indicate a need to retry an authentication check
-		if apierrors.IsServerTimeout(err) || apierrors.IsTimeout(err) || apierrors.IsTooManyRequests(err) {
```

Comment:
why did you remove `IsServerTimeout`?

Reply:
Because it's already checked in `SuggestsClientDelay`. I can return it, if you want the check to be more explicit